disparate-flies/WorkforceManagement
# Computer Controller - View and Create

Please structure your pull requests this way:

- **Purpose of PR**: User is now able to click on the computer tab inside navbar. User is now able to see a list of computers. User can click on the Make of the computer in order to see the details. User can click on the Create New link and add a computer.

- **Issue number this request impacts**: #13 & #15 

- **Description of what was changed**: Created the Computer Controller & Views - Made Edit on Computer Model - Deactivated the hyperlinks on Edit and Details with in the Index View

1. Computer Controller(wrote code for the GET-DETAILS-CREATE)
2. Computer Model(removed the required on the Decommission Date and the Condition - this is not required for the create so wouldn't validate the state of model)
3. Computer View folder with Index-Details-Create razor view pages 
4. Index View shows only text for the Edit and Details since the links aren't being used

- **Thorough list of steps to test**
1. checkout master
2. git fetch
3. cd into my branch (CompController)
4. type start WorkforceManagement.sln inside your terminal, it will open up in Visual Studio
5. open the Computer Controller inside the file 
6. press green play button in the navbar of VS 
7. terminal will open telling you to go into browser and enter localhost:5000 
8. click on the computer tab inside the navbar
9. You should see a list of all computers by their Id, Manufacturer, and Make
10. you should not be able to click on the words Edit and Details since they are not being used and have been turned to text
11. click on the Make, the details of the specified computer should be displayed
12. go back to previous page
13. click the Create New link above the list of computers
14. Insert a Purchase Date, Manufacturer, and Make of a new computer
15. page should redirect back home and the new computer should display at the bottom of the list


if you don't do this you will be
**FIRED**
